keyboards/handwired/sauceball/info.json

```diff
@@ -0,0 +1,38 @@
+{
+    "manufacturer": "xia0",
+    "keyboard_name": "sauceball",
+    "maintainer": "xia0",
+    "bootloader": "atmel-dfu",
```

Comment:
If your hardware section is talking about teensy vs pro micro, this is then incorrect.

Reply:
```suggestion
    "bootloader": "halfkay",
```